src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const delProp of action.keysToRemove) {
```

Comment:
you should try to make your code maximally clear
```suggestion
        for (const propToDelete of action.keysToRemove) {
```